README.md

```diff
@@ -12,6 +12,8 @@ To send an order to the DEX, a user needs to send a regular transfer transaction
 - **Close order**: `${targetChain},close,${orderId}`
 - **Credit**: `credit`
 
+- **orderId** is a transaction Id.
```

Comment:
Maybe we should move this down further (maybe incorporate it into the new paragraph you added) because here it looks like `orderId` is an action, but it's just a parameter.

Reply:
Or you could add a new subheading here called `Parameters` and define each of the parameters such as `targetChain`, `bidOrAskPrice`, ..., `orderId`, ...